tools/klee/main.cpp

```diff
@@ -606,11 +606,13 @@ std::string KleeHandler::getRunTimeLibraryPath(const char *argv0) {
 
   SmallString<128> libDir;
 
-  if ( strcmp(toolRoot.c_str(), KLEE_INSTALL_BIN_DIR ) == 0)
+  if (toolRoot.str().endswith( KLEE_INSTALL_BIN_DIR ))
   {
```

Comment:
Please add a check that KLEE_INSTALL_BIN_DIR is not length 0
The same for KLEE_INSTALL_RUNTIME_DIR.